### ISSUE TITLE
Can't remove dependencies from a task

### ISSUE THREAD

**Issue by yasledesma** [GothenburgBitFactory/taskwarrior#3634]
#### To report a bug...
* What command(s) did you run?
```
task <id> modify depends:
```

* What did you expect to happen?
I was expecting the command to remove all the dependencies I had previously added to the task.

* What actually happened?
The dependencies were not removed from the task. But it still printed a success message, and I had to proceed to edit out the dependency:
```
Modifying task <id> <task>.
Modified 1 task.
Project <project> is 55% complete (9 of 20 tasks remaining).
```

* Paste the output of the ```task diag``` command.
```
task 2.6.1
   Platform: Linux

Compiler
    Version: 11.2.0
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++17

Build Features
      CMake: 3.18.4
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.7.1
 Build type: None

Configuration
       File: /home/user/.taskrc (found), 1779 bytes, mode 100664
       Data: /home/user/.task (found), dir, mode 40755
    Locking: Enabled
         GC: Enabled
  rc.editor: nvim
     Server:
         CA: -
Certificate: -
        Key: -
      Trust: strict
    Ciphers: NORMAL
      Creds:

Hooks
     System: Enabled
   Location: /home/user/.task/hooks
             (-none-)

Tests
   Terminal: 210x47
       Dups: Scanned 82 tasks for duplicate UUIDs:
             No duplicates found
 Broken ref: Scanned 82 tasks for broken references:
             No broken references found
```

I think this is probably a regression, because it has been reported before: https://github.com/GothenburgBitFactory/taskwarrior/issues/2655

**Comment by djmitche** [comment]
The linked bug appears to have been fixed in 2.6.2, but the `task diag` says 2.6.1 I'd suggest updating to at least 2.6.2, or [to 3.0](https://taskwarrior.org/docs/upgrade-3/)

**Comment by djmitche** [comment]
(I'm happy to re-open if this doesn't address the issue)

**Comment by yasledesma** [comment]
> The linked bug appears to have been fixed in 2.6.2, but the `task diag` says 2.6.1 I'd suggest updating to at least 2.6.2, or [to 3.0](https://taskwarrior.org/docs/upgrade-3/)

Yes, it works now in 3.1.0! The release in the download page (task-3.0.2.tar.gz) fails to build, but building from the repo works just fine. Thank you!

**Comment by djmitche** [comment]
Fixing that download page issue is https://github.com/GothenburgBitFactory/tw.org/issues/963
